R/emtr.R

```diff
@@ -1,10 +1,50 @@
 parameters_from_file <- function(parameters_file) {
-  parameters_df <- openxlsx::read.xlsx(parameters_file)
-  parameters <- parameters_from_df(parameters_df)
+  params_type <- tools::file_ext(parameters_file)
+  if (params_type %in% c("xlsx", "xls")) {
+    parameters_df <- openxlsx::read.xlsx(parameters_file, sheet = 1) %>% as.data.frame()
+    parameters <- parameters_from_df(parameters_df)
+  } else if (params_type %in% c("yaml", "yml")) {
+    parameters <- yaml::read_yaml(parameters_file)
+  } else {
+    error_msg <- paste0("Unsupported params type: ", params_type)
+    stop(error_msg)
+  }
+  # Rename any missing app parameters if possible
+  renamed_app_params <- c(

```

Comment:
Why not just make the parameters in their correct name when we prepare the app parameters? So we can save this step.

Reply:
I would like to keep this so that we can upload TAWA parameters and have them be converted automatically; then we don't always have to prepare app parameters as a separate step.